sdk/openai/Azure.AI.OpenAI/src/Generated/OpenAIClient.cs

```diff
@@ -115,11 +112,6 @@ internal virtual async Task<Operation<BinaryData>> BeginAzureBatchImageGeneratio
         /// This <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
         /// </description>
         /// </item>
-        /// <item>
```

Comment:
why does this comment go away without the method going away?  If they added it in custom code we should honor that and keep this comment as it helps drive people to the more convenient overload.

Reply:
This message is removed by my change.
When this protocol method is generated as public, and the corresponding convenience method is made internal because of some low confident stuff, we will remove this line.
I would be OK if we keep this message.
Currently the generator will not pick up methods in customized code - therefore currently we cannot determine when to write this and when not to.
We could improve this later I believe

Reply:
Offline discussed that this is not a big issue because they actually write a new method as convenience method, and the corresponding protocol method is internal now.